bin/build-phar.sh

```diff
@@ -1,37 +1,6 @@
 #!/usr/bin/env bash
-if ! php -r 'extension_loaded("bz2") or exit(1);' ; then
-	echo "You need to install (or enable) bz2 php extension"
-	exit 1
-fi
+composer bin box install
 
-composer global require 'humbug/php-scoper:^1.0@dev'
-composer global require 'humbug/box:3.0.0-alpha.0'
-composer install --no-dev
-[ -d build ] || mkdir build
-[ -d build/psalm ] || mkdir build/psalm
-# increase FD limit, or Phar compression will fail
-ulimit -Sn 4096
+vendor/bin/box compile
 
-rm -f bin/psalm.phar
-
-# Prefixes the code to be bundled
-php -d memory_limit=-1 `which php-scoper` add-prefix --prefix='PsalmPhar' --output-dir=build/psalm --force
```

Comment:
the scoping is done via Box now, the only issue is https://github.com/humbug/php-scoper/issues/178 so right now the prefix is really random

Reply:
the memory is also bumped by Box although the number is quite arbitrary so it may need more

Reply:
The deterministic prefix was on purpose to enable better cache re-use, but I don't really mind caches being busted between new phars.